src/components/OrganizationSaas/Dashboard/EditProfile.tsx

```diff
@@ -0,0 +1,243 @@
+  OrganzationTypesSelector,
+} from '~components/Layout/SaasSelector'
+import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
+import fallback from '/assets/default-avatar.png'
+
+interface EditFormFields {
+  name: string
+  description: string
+  avatar: string
+}
+
+const REGEX_AVATAR = /^(https?:\/\/|ipfs:\/\/)/i
+
+const EditProfile = () => {
+  const { textColor, textColorSecondary, bgSecondary } = useDarkMode()
+  const { account, fetchAccount } = useClient()
+  const { update } = useOrganization()
+  const { t } = useTranslation()
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+
+  const required = {
+    value: true,
+    message: t('form.error.field_is_required'),
+  }
+
+  const methods = useForm({
+    defaultValues: {
+      name: account?.account.name.default || '',
+      description: account?.account.description.default || '',
+      avatar: account?.account.avatar || '',
+    },
+  })
+
+  const {
+    watch,
+    setValue,
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = methods
+
+  const avatar = watch('avatar')
+
+  const correctAvatarFormat = (val: string) => REGEX_AVATAR.test(val)
+
+  const onSubmit: SubmitHandler<EditFormFields> = async (values: EditFormFields) => {
+    setLoading(true)
+
+    try {
+      await update(new Account({ ...account?.account, ...values }))
+      fetchAccount()
+    } catch (err: any) {
+      setError(errorToString(err))
+    } finally {
+      setLoading(false)
+    }
+  }
+
+  return (
+    <FormProvider {...methods}>
+      <Box height='100%' maxH='100%' overflowY='auto'>
+        <Flex as='form' flexDirection='column' gap={12} maxW='600px' mx='auto'>
+          <Flex flexDirection='column' gap={6}>
+            <Box>
+              <Text color={textColor} fontWeight='bold'>
+                {t('edit.public_org_info', { defaultValue: 'Public Organization Information' })}
+              </Text>
+              <Text color={textColorSecondary} fontSize='sm'>
+                {t('edit.public_org_info_description', {
+                  defaultValue:
+                    'If your organization already have a profile, ask the admin to invite you to your organization',
+                })}
+              </Text>
+            </Box>
+            <InputCustom
+              formValue='name'
+              label={t('name', { defaultValue: 'Name' })}
+              placeholder={t('form.account_create.title_placeholder', {
+                defaultValue: "Enter your organization's emial",
+              })}
+              type='text'
+              required
+            />
+            <InputCustom
+              formValue='website'
+              label={t('website', { defaultValue: 'Website' })}
+              placeholder={t('form.account_create.website_placeholder', {
+                defaultValue: 'https://example.com',
+              })}
+              type='text'
+              required
+            />
```

Comment:
Same here, this input exists on another form, so extract it to the `src/components/Layout/InputCustom.tsx` file and import it from there. Check all others that need this changes

Reply:
31b0ad5